--- a/linux/main.cc	2020-03-04 15:26:37.621209930 +0100
+++ b/linux/main.cc	2020-03-04 14:41:36.700815865 +0100
@@ -45,6 +45,10 @@
 
   // Arguments for the Flutter Engine.
   std::vector<std::string> arguments;
+#ifdef NDEBUG
+    arguments.push_back("--disable-dart-asserts");
+    arguments.push_back("--disable-observatory");
+#endif
 
   flutter::FlutterWindowController flutter_controller(icu_data_path);
   flutter::WindowProperties window_properties = {};
