--- a/lib/main.dart	2020-03-04 15:26:08.964892704 +0100
+++ b/lib/main.dart	2020-03-04 11:34:41.313012581 +0100
@@ -12,19 +12,6 @@
 
 import 'gallery/app.dart';
 
-// Sets a platform override for desktop to avoid exceptions. See
-// https://flutter.dev/desktop#target-platform-override for more info.
-// TODO(gspencergoog): Remove once TargetPlatform includes all desktop platforms.
-// This is only included in the Gallery because Flutter's testing infrastructure
-// uses the Gallery for various tests, and this allows us to test on desktop
-// platforms that aren't yet supported in TargetPlatform.
-void _enablePlatformOverrideForDesktop() {
-  if (!kIsWeb && (Platform.isWindows || Platform.isLinux)) {
-    debugDefaultTargetPlatformOverride = TargetPlatform.fuchsia;
-  }
-}
-
 void main() {
-  _enablePlatformOverrideForDesktop();
   runApp(const GalleryApp());
 }
